Assets/Scripts/Core/GridCombatSystem.cs

```diff
@@ -193,5 +195,41 @@ private enum State {
 		public CoreUnit GetActiveUnit() {
 			return unitGridCombat_;
 		}
+
+		// It is possible to add another float variable WeaponModifierHitChance to this method, then adjust float hitChance accordingly.

```

Comment:
This is not a good comment as it does not explain the use of this function here.